### PR TITLE
Add sqlite database

### PR DESCRIPTION
**Pull request by eliaskioni**
Add  sqlite schema database if DATABASE_URL is not found in the environment
Reason if to make easier to run django_airflow for users without postgres installed

### CODE REVIEW

**Comment by mwaaas** [comment]
This design looks much better. 
`DATABASES = {
    'default': dj_database_url.config(
        default=os.environ.get('DATABASE_URL', "sqlite:////ussd_airflow.sqlite")
    )
}`

**Comment by mwaaas** [comment]
@eliaskioni  Did you try the recommended choices

**Comment by eliaskioni** [comment]
@mwaaas have made the change.Thanks

File: ussd_airflow/settings.py [hunk 1]
```diff
@@ -81,11 +81,9 @@
 # https://docs.djangoproject.com/en/1.9/ref/settings/#databases
 DATABASES = {
     'default': dj_database_url.config(
-        default=os.environ.get('DATABASE_URL')
+          default=os.environ.get('DATABASE_URL', 'sqlite:////{}UssdAirflow.sqlite3'.format(BASE_DIR))
```
Comment (eliaskioni) [inline ussd_airflow/settings.py]: Its an error. Have fixed

**Pull request merged**